### ISSUE TITLE
[Bug] Left Navigation Doesn't Work in Chrome and Opera on Linux

### ISSUE THREAD

**Issue by markizano** [grafana/grafana#8389]
Grafana: 4.2.0
Datasource: Graphite-0.9.12
Devuan/Debian Linux: 4.8.0-0.bpo.2-amd64

## What did you do?
Use the left navigation to reach API keys, Org Users, or Alerting List - any of the other places to visit.
![nav](https://cloud.githubusercontent.com/assets/360451/26107795/245a3e42-3a10-11e7-8c43-b2bd0e763742.png)

## What was the expected result?
Successful navigation to the selected page.

## What was the actual result?
In some cases nothing, as if there was no href= attached to the <a> anchor.
In other cases, it would navigate me to some arbitrary graph.

## Browser Details
FireFox/Linux-45.8.0 - Works!
Chrome/Linux 57.0.2987.110 - Does NOT work.
Opera/Linux 44.0.2510.857 - Does NOT work.
FireFox/Windows-42.0 - Works!
Chrome/Windows 58.0.3029.110 - Works!
Opera/Windows 45.0 - Works!

Upon closer inspection on how the page behaves, it's as if the on-click or on-mousedown event that is presiding over the <aside /> menu is overriding the <a> click event to navigate.
When I click/mouse-down, I see a flash of the submenu to make it reappear. This may be conflicting with the natural click on the link.

I don't have the UI prowess to look into this, but it's really frustrating as I have to copy the link location and manually navigate instead of being able to freely click.

**Comment by torkelo** [comment]
very strange, using chrome 57 on linux and can't replicate any of the behaviors you describe. All the links works like normal (left click or right click open in tab) works as expected. 

Maybe some chrome plugin that your running?

**Comment by markizano** [comment]
Thanks for testing, I don't have another linux client to test this on, so I wasn't sure if it was just me.
I tried again in an incognito window where all my plugins are disabled and still don't navigate cleanly.
Here's a video clip to hopefully show what I'm seeing:
https://markizano.net/misc/grafana.mp4

**Comment by torkelo** [comment]
anything in chrome developer console (js errors)

**Comment by markizano** [comment]
Negative.
Its as if the click event isn't being invoked.
No new Network tab entries, and no new Console entries.

**Comment by daniellee** [comment]
Just as a test, are you seeing the same symptoms on our [demo site](http://play.grafana.org)?

**Comment by markizano** [comment]
I am, actually.

While I was making this video clip for the demo site, I noticed something:
https://markizano.net/misc/grafana-chrome.mp4
(For this incognito browser session, I had previously logged into Github to avoid the authentication page when clicking "Login with Github")

I notice the first time I click on something in a sub-menu, it will navigate, but on subsequent navigations it fails.
If I make it refresh the page by using one of the top-level navigation menus, then use a sub-menu, it repeats: The first navigation works, but subsequent navigations on a sub-menu don't appear to be working.

Are either of you seeing this kind of behaviour now that we have this detail?
Here's that clicking again with the developer console up:
https://markizano.net/misc/grafana-webdev.mp4

**Comment by torkelo** [comment]
very strange, unable to replicate this on any chrome browser or OS :(

**Comment by daniellee** [comment]
I can't either. Works as it should in Chrome (58.0.3029.110) and Firefox (53.0.2) on Ubuntu 16.04. 

- Can you ask a colleague to test with their Chrome in incognito mode?
- Any proxy or Adblocker that could be blocking the JavaScript?

**Comment by markizano** [comment]
ok, I asked someone to try again on another workstation. On ubuntu16 and chrome51, he wasn't able to reproduce the issues seen.

I've got one more person I know with a linux workstation on IST, he will try during his daytime.
I'll report his findings and we'll determine the fate of this bug on that.

I was using an incognito tab to prevent any extensions from interfering with the results of the test.
Additionally, just to be sure it wasn't adBlock working against me, I whitelisted the *grafana.org/* domain.
No outbound HTTP proxy, as I'm connected to the network directly from home.
